src/lobsterpy/plotting/__init__.py

```diff
@@ -1093,6 +1093,18 @@ class IcohpDistancePlotter:
             Defaults to False for COHPs.
     """
 
+    COLOR_PALETTE = [
```

Comment:
what do you think about a customizable color palette?

Reply:
Is it there, in get_plot one can pass list of custom hex colors and it will use that then 😄

Reply:
cool, didn't see it x)